changelogs/template.md

```diff
@@ -0,0 +1,26 @@
+# EMAP Release
+
+**Date: __TODO: FIXME__ Changes in this release**
+
+---
+
```

Comment:
The template should probably have the table of changed attributes so we don't have to look at an old one and copy it